sherpa/astro/__init__.py

```diff
@@ -17,4 +17,5 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-# Nothing in here yet
+hc = 12.39841874  # nist.gov in [keV-Angstrom]
```

Comment:
Can we add a docstring after these rather than a comment - e.g.

```
hc = silly-number
"""The product of h * c using values from nist.gov appropriate for converting between keV and Angstroms."""
```

(but better written).

We will then get this included in the documentation if we ever add sherpa.astro to the documentation. Which we probably should.